build.gradle.kts

```diff
@@ -38,6 +40,11 @@ dependencies {
 
     implementation("io.github.microutils:kotlin-logging:$kotlinLoggingVersion")
 
+    implementation("org.apache.logging.log4j:log4j-api:$log4j2Version")
+    implementation("org.apache.logging.log4j:log4j-core:$log4j2Version")
+    implementation("org.apache.logging.log4j:log4j-slf4j-impl:$log4j2Version")
+    implementation("com.vlkan.log4j2:log4j2-logstash-layout-fatjar:0.15")
```

Comment:
mknor: we'll need a log4j2.xml  to configure logstash log - see https://github.com/navikt/dagpenger-joark-mottak/blob/master/src/main/resources/log4j2.xml (To have thew logs visible in kibana)